lib/core-events/src/index.ts

```diff
@@ -6,10 +6,6 @@ enum events {
   CURRENT_STORY_WAS_SET = 'currentStoryWasSet',
   // Emitted by the preview whenever the list of stories changes (in batches)
   SET_STORIES = 'setStories',
-  // Emitted by the preview when the current story (or error) should be rendered
-  RENDER_CURRENT_STORY = 'renderCurrentStory',
-  // Deprecated version of the above
-  STORY_RENDER = 'storyRender',
```

Comment:
This is a major breaking change for addons, we should document in migration.md?

Reply:
OK!

Reply:
Done: https://github.com/storybookjs/storybook/pull/11080/commits/ea86077196a0b8ae20ba00169a1ac0623213e394